futures-executor/src/local_pool.rs

```diff
@@ -192,17 +191,9 @@ impl Default for LocalPool {
     }
 }
 
-lazy_static! {
-    static ref GLOBAL_POOL: ThreadPool = ThreadPool::builder()
```

Comment:
huh-- too bad this didn't generate a warning :(

Reply:
@cramertj `lazy_static` applies `#[allow(dead_code)]` to the code: https://github.com/rust-lang-nursery/lazy-static.rs/blob/1.3.0/src/lib.rs#L153